apps/site/sentry.client.config.ts

```diff
@@ -9,12 +9,13 @@ Sentry.init({
   dsn,
   enabled: !!dsn,
   environment: process.env.NEXT_PUBLIC_VERCEL_ENV ?? "unset",
-  integrations: process.env.NEXT_PUBLIC_SENTRY_REPLAYS_SAMPLING_RATE
+  integrations: process.env.NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLING_RATE
```

Comment:
Aligns with `replaysSamplingRate` being renamed to `sessionSampleRate`. Old key generates a console warning.

I've already added the new var to Vercel. We'll delete the old one after merging.